test/JDBC/upgrade/master/schedule

```diff
@@ -343,4 +343,5 @@ BABEL-4175
 BABEL_4330
 TestDatatypeAggSort
 babel_index_nulls_order
+drop_index-before-15_5-or-16_2
```

Comment:
This should just be `drop_index`

Reply:
that was a separate PR, already merged earlier